### PR TITLE
Check if image is not default image - Hooks

### PR DESCRIPTION
**Pull request by guyfreund**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/18704

## Description
Added hook test to check if the image is not the default image

## Must have
- [x] Tests
- [x] Code Review

### CODE REVIEW

File: Tests/scripts/hook_validations/image.py [hunk 1]
```diff
@@ -86,3 +90,45 @@ def is_existing_image(self):
             return False
 
         return True
+
+    def is_not_default_image(self):
+        """Check if the image is the default one"""
+        if re.match(IMAGE_REGEX, self.file_path, re.IGNORECASE):
```
Comment (Itay4) [inline Tests/scripts/hook_validations/image.py]: add comment what this condition handles

File: Tests/scripts/hook_validations/image.py [hunk 2]
```diff
@@ -86,3 +90,45 @@ def is_existing_image(self):
...
+                    encoded_image = encoded_image.decode("utf-8")
+                if encoded_image == DEFAULT_IMAGE_BASE64:  # disable-secrets-detection
+                    print_error("{} is the default image, please change to the "
+                                "integration image.".format(self.file_path))
+                    self._is_valid = False
+                    return False
+                else:
+                    return True
+
+        else:
```
Comment (Itay4) [inline Tests/scripts/hook_validations/image.py]: and add comment what this condition handles

File: Tests/scripts/hook_validations/image.py (hunk 1 above)
Comment (guyfreund) [reply Tests/scripts/hook_validations/image.py]: fixed

File: Tests/scripts/hook_validations/image.py (hunk 2 above)
Comment (guyfreund) [reply Tests/scripts/hook_validations/image.py]: fixed

**Pull request merged**